TheWarWithin/MonkBrewmaster.lua

```diff
@@ -1042,6 +1044,59 @@ spec:RegisterStateExpr( "boc_count", function()
     return blackoutComboCount
 end )
 
+-- Snapshotting
+
+local bocConsumptionTime = 0 -- time at which Blackout Combo was most recently consumed
```

Comment:
Is this logic used in default APLs (or APLs from your addon)?

Reply:
This is used in APLs from my addon. In the Equinox Brewmaster monk guide, he recommends *"upkeeping the 10% DR on Breath of Fire with Blackout Combo"*, and there was no way to track that in Hekili. I dug around in SimulationCraft and saw that they actually have expressions for checking this but are unused in the current APLs. Those expressions are what I've added to Hekili to keep APL language in sync with SimC.